sass/_SearchInterface.scss

```diff
@@ -9,8 +9,6 @@
   font-family: 'Lato', $helvetica, sans-serif;
   font-size: 15px;
   color: $color-dark-grey;
-
-  .coveo-highlight,
   .highlight {
```

Comment:
The jira specifies only the `coveo-highlight` class, but what about `.highlight`? Is this class being used?

Reply:
Moving it outside/globally would be a huge mistake. It would pretty much mess up a ton of existing client website.

.coveo-highlight is the one used for query suggestion.